packages/nartus_ui_package/lib/theme/nartus_theme.dart

```diff
@@ -1,9 +1,15 @@
 import 'package:flutter/material.dart';
 
 import 'nartus_color.dart';
+import 'nartus_text_theme.dart';
+import 'package:google_fonts/google_fonts.dart';
 
 ThemeData lightTheme = ThemeData(
   primarySwatch: lightColorSwatch,
```

Comment:
you can remove primarySwatch now, as we have our colorScheme now

Reply:
@suesitran Done, i have pushed the update